### DISCUSSION TITLE
React JSON Schema Form binding for Primer

### DISCUSSION THREAD

**Discussion by echarles** [primer/react#3152] [Show and tell]
Hi, just wanted to share some work we are doing to add Primer binding to React JSON Schema Form solution.

The repo is available on https://github.com/datalayer/primer-rjsf and provides today a complete coverage of the RJSF API. Playground on https://datalayer.github.io/primer-rjsf

Hope this can be useful. Also happy to get any feedback, and be listed as external resources on Primer documentation if there is a place for that.

**Comment by echarles** [comment]
We miss a Slider to implement the RangeWidget, so we picked the MUI one for now https://mui.com/material-ui/react-slider/

**Comment by lesliecdubs** [comment]
Thanks for sharing @echarles!

@siddharthkp as our First Responder this week, could you take a look and weigh in with any feedback? 

Also tagging in @emilybrick to look into whether this might be a good fit to link to as an external resource from the Primer docs.

**Comment by siddharthkp** [comment]
Hi! 

This is a really interesting project! I wasn't aware of it till now. Will play around with it :)
